lib/menu.rb

```diff
@@ -0,0 +1,25 @@
+
+
+class Menu
+attr_reader :items
+  def initialize
+    @items = [{:pizza => "Margherita", :price =>  9.99}, 
+              {:pizza => "Neapolitan", :price => 12.99},
+              {:pizza => "Greek", :price => 13.99},
+              {:pizza => "Padana", :price => 11.99}]
+  end
+
+  def view
```

Comment:
A pretty common pattern in OOD is to separate your business logic (anything dealing with data or state) from presentation logic (anything dealing with displaying information to human users).

Right now this method is dealing with presentation logic where as the rest of the class is dealing with business logic - there could be a good argument for extracting out a `Printer` class or something similar